File: Python/sorts/insertion_sort.py
import random
import datetime

x = []

for i in range(0, 100):
    randomNumber = int(round(random.random() * 10000))
    x.append(randomNumber)

startTime = datetime.datetime.now()

def insertionSort(list):
    for index in range(0, len(list)):
        element = list[index]
        j = index
        while j > 0 and element < list[j-1]:
            list[j] = list[j - 1]
            j -= 1
        list[j] = element

    timeElapsedInSeconds = (datetime.datetime.now() - startTime).total_seconds()
    print "time elapsed in seconds:", timeElapsedInSeconds

    return list



print insertionSort(x)

File: Javascript/Sorts/insertion_sort.js
var arr = [1, 8, 9, 5, 3, 2];

function insertion_sort(arr) {
    for (var i = 0; i < arr.length; i++) {
        var element = arr[i];
        var j = i;
        while (j > 0 && arr[j-1] > element) {
            arr[j] = arr[j - 1];
            j--;
        }
        arr[j] = element;
    }
    console.log(arr);
    return arr;
}

insertion_sort(arr);